doc/source/contributing.rst

```diff
@@ -17,42 +17,55 @@ Contribute Code or Documentation
 --------------------------------
 **Your contributions to SNEWPY are welcome!**
 
-To contribute, please clone the GitHub repository::
+To contribute, please `create your own fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_
+of the SNEWPY repository [#fn_fork]_, then clone it::
 
-    git clone git@github.com:SNEWS2/snewpy.git
+    git clone git@github.com:<YOUR-USERNAME>/snewpy.git
     cd snewpy/
 
-then make your changes and install the package with the dependencies for the development::
-    
-    pip install ".[dev]"
-
-or, if you want to build the documentation::
-
-    pip install ".[docs]"
-    cd doc/
-    make html
-
+Next, make your changes and try them out. Where relevant, run tests or build
+documentation as described in the following subsections.
 Once you're happy with your changes, please 
-`submit a pull request <https://github.com/SNEWS2/snewpy/pulls>`_.
+`create a pull request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork>`_.
 If you plan larger changes, it’s probably a good idea to open an issue first
 to coordinate our work.
 
-Testing
-~~~~~~~
+Running Tests
+~~~~~~~~~~~~~
+
+SNEWPY uses the `pytest <https://docs.pytest.org>`_ package for automated testing.
+First, to install the necessary packages, use::
 
-SNEWPY uses the `pytest <https://docs.pytest.org>`_ package
-for automated testing. Tests will run when you submit a pull request
-or you can run them manually using::
+    pip install ".[dev]"
+
+You can then run the tests using the
 
     pytest
 
-command in the SNEWPY root directory.
-SNOwGLoBES interface tests requires ``$SNOWGLOBES`` environment variable to be set to the path of you SNOwGLoBES installation.
-If you want to skip this part, use::
-    
-    pytest -k 'not snowglobes'
+command in the SNEWPY root directory. To skip integration tests which depend
+on SNOwGLoBES---or to run _only_ those tests---you can use one of::
```

Comment:
This looks like Markdown and doesn't render correctly in RST. I suggest:
```suggestion
on SNOwGLoBES (or to run *only* those tests) you can use one of::
```

Reply:
Indeed; too much switching back and forth between README.md and contributing.rst as I was editing that … 😅 Thanks, good catch!